Makefile

```diff
@@ -0,0 +1,24 @@
+PROTO_TOP_DIR := $(shell cd ../bucketeer && pwd)
+PROTO_FOLDERS := event/client feature gateway user
+PROTO_OUTPUT := proto_output
+IMPORT_PATH_FROM := github.com/ca-dp/bucketeer
+IMPORT_PATH_TO := github.com/ca-dp/bucketeer-go-sdk
+
+.PHONY: copy-protos
+copy-protos: .gen-protos
+	rm -rf proto
```

Comment:
Are you going to use bazel to build bucketeer-go-sdk?
If so, it causes a problem because each proto directory includes BUILD.bazel file.

Reply:
Let me confirm this and get back to you.

Reply:
@mura-s I confirmed, Hiro is not going to use bazel for this project.